penguin/tools/core/support.py

```
@@ -1763,6 +1731,11 @@ def apply_unified_patch(
                 return msg
         else:
             # Apply patch to existing file using robust single-file editor
+            if target.exists() and str(target) not in snapshots:
+                try:
+                    snapshots[str(target)] = target.read_bytes()
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)